docs/en/getting_started.md

```diff
@@ -174,65 +178,99 @@ python tools/mview_mperson_evaluation.py \
 
 #### Learning-based methods
 
-For learning-based methods, more details about dataset preparation, model weights and checkpoints download can be found at [evaluation tutorial](./tools/eval_model.md).
+1. Download and install the `Deformable` package (Skip if you have done this step before)
 
-With the downloaded pretrained MvP models from [model_zoo](./benchmark.md):
+Run the script:
 
-```shell
-sh ./scripts/val_mvp.sh ${NUM_GPUS} ${CFG_FILE} ${MODEL_PATH}
+```bash
+sh scripts/download_install_deformable.sh
 ```
 
-Example:
-```shell
-sh ./scripts/val_mvp.sh 8 configs/mvp/shelf_config/mvp_shelf.py weight/xrmocap_mvp_shelf.pth.tar
+2. Download dataset and pretrained model, taking Shelf dataset as an example:
+
+```bash
+# download Shelf dataset
```

Comment:
What about `# download Shelf dataset (16G)`?
Knowing the dataset size may encourage users to set aside some time for data downloading.

Reply:
Added in [bb71f8e](https://github.com/openxrlab/xrmocap/pull/29/commits/bb71f8eef16e36d321b7ee98c8689cb11bdda78c).